src/main/java/org/example/servlets/FeedbackAddServlet.java

```diff
@@ -0,0 +1,60 @@
+package org.example.servlets;
+
+import org.example.HibernateUtil;
+import org.example.entity.Advertisement;
+import org.example.entity.Feedback;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static java.lang.Integer.parseInt;
+@WebServlet("/fb")
+public class FeedbackAddServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/WEB-INF/jsp/feedback.jsp").forward(req, resp);
+    }
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        Feedback feedback = new Feedback();
+        try {
+            SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+            Session session = sessionFactory.openSession();
+            Transaction t = session.beginTransaction();
+            int paramId = session.load(Integer.class, req.getParameter("adId"));
```

Comment:
Actually, we couldn't get Integer from the database.
Also nased on your setter, this logic doesn't work at all